float64 motor1pos
float64 motor2pos